docs/view_mobie_zarr.rst

```diff
@@ -0,0 +1,43 @@
+View zarr file using MoBIE and BDV
+==================================
+
+Description
+-----------
+
+This section shows how to view ome.zarr files in Fiji using `MoBIE <https://github.com/mobie/mobie-viewer-fiji)>`__.
+
+We show:
+
+- how to install the required dependencies
+- how to view an ome.zarr file stored in S3 using the scripting editor of Fiji.
+
+Setup
+-----
+
+- Install Fiji on a local machine.
+- Activate the ``MoBIE-beta`` update site.
```

Comment:
Maybe a question for @tischi since he had mentioned using the non-beta version.

See https://omero-guides--22.org.readthedocs.build/projects/fiji/en/22/view_mobie_zarr.html for a rendered version.

Reply:
I could not open the image using the non-beta update-site cc @tischi
unless I had an old jar kicking around. 
I will double-check and update accordingly

Reply:
I must have had a old jar in the mix preventing MoBIE to work
I will remove the beta option
@tischi, when using the non-beta I have a warning
```
[WARNING] Ignoring unsupported output: result [bdv.util.BdvStackSource]
```